MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -591,6 +591,14 @@ private MetadataPluginSummary GetPluginSummary<T>()
                 Type = MetadataPluginType.MediaSegmentProvider
             }));
 
+            // Filesystem to metadata resolvers
+            var metadataResolverProviders = summary.Plugins.Select(p => p.Type == MetadataPluginType.MetadataResolver && p.Name != null ? p : null);
```

Comment:
Do we want to tack on a `.Where(i != null)` here?

Reply:
how _could_ they ever be null?

Reply:
I was referring to the adding onto the block `.Select(p => p.Type == MetadataPluginType.MetadataResolver && p.Name != null ? p : null)`, where what comes out of the `.Select` could be null if the `.Type` is not `MetadataPluginType.MetadataResolver` OR if it has a `null` setting on the `.Name`... in those cases we would get a `null` out of the ternary... so maybe we should filter/skip those?

`.Select(p => p.Type == MetadataPluginType.MetadataResolver && p.Name != null ? p : null).Where(n => n != null);`